### PR TITLE
Deployment manager stats + source field

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsHtmlSnippet.scala [hunk 1]
```diff
@@ -12,25 +14,68 @@ case class UsageStatisticsHtmlSnippet(value: String)
...
+                                             processingTypeStatisticsMap: Map[ProcessingType, ProcessingTypeUsageStatistics]): ListMap[String, String] = {
+    val deploymentManagerTypes = processingTypeStatisticsMap.values.map(_.deploymentManagerType).map {
+      case dm if knownDeploymentManagerTypes.contains(dm) => dm
+      case _ => aggregateForCustomValues
+    }
+    val dmParams = prepareValuesParams(deploymentManagerTypes, "dm")
+
+    val processingModes = processingTypeStatisticsMap.values.collect {
+      case ProcessingTypeUsageStatistics(_, Some(mode)) if knownProcessingModes.contains(mode) => mode
+      case ProcessingTypeUsageStatistics(_, Some(_)) => aggregateForCustomValues
```
Comment (mproch) [inline designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsHtmlSnippet.scala:47]: I think this should be default at the end, regardless of mode?

File: nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh [hunk 2]
```diff
@@ -35,6 +35,10 @@ else
   JAVA_PROMETHEUS_OPTS="-javaagent:$agentPath=$PROMETHEUS_METRICS_PORT:$PROMETHEUS_AGENT_CONFIG_FILE"
 fi
 
+if [ "$USAGE_REPORTS_SOURCE" == "" ]; then
```
Comment (mproch) [inline nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh:38]: I wonder if we need to allow overrinding it? Or it's just in case?

Comment (arkadius) [reply nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh:38]: It is intensional. "docker" is too much general. I want to override it by "quickstart-docker-compose", "helmchart"

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsHtmlSnippet.scala (hunk 1 above)
Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsHtmlSnippet.scala]: :+1: changed

File: nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh (hunk 2 above)
Comment (mproch) [reply nussknacker-dist/src/universal/bin/nussknacker-entrypoint.sh:38]: :+1:

**Review by mproch — APPROVED** [review]
:+1:

**Pull request merged**